openwisp_monitoring/device/migrations/0003_remove_monitoring_templates.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.1.2 on 2020-12-20 06:15
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def delete_monitoring_templates(apps, schema_editor):
+    Template = apps.get_model('config', 'Template')
+
+    Template.objects.filter(
+        Q(config__contains='/usr/sbin/openwisp-monitoring')
+        & Q(config__contains='/usr/sbin/netjson-monitoring'),
+    ).delete()
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('device_monitoring', '0002_create_template'),
+    ]
+
+    operations = [
+        migrations.RunPython(delete_monitoring_templates, reverse_code=migrations.RunPython.noop)
+    ]
```

Comment:
let's not do this. Let's instead change the data migrations which introduce the template so they do nothing. Otherwise we risk that some people will upgrade the system and not read this, get their monitoring template removed without having installed the new package.

We should advise users using previous versions to remove this template manually and install the new openwrt package in the README.

Reply:
> We should advise users using previous versions to remove this template manually and install the new openwrt package in the README.

Sounds good!

Reply:
>Let's instead change the data migrations which introduce the template so they do nothing.

What will be the best way to do so? Should I remove that function or just empty the operations list?

Reply:
Replacing the function with `migrations.RunPython.noop` will be good I guess. Don't forget to leave a comment explaining why migration file is like that.

> Should I remove that function or just empty the operations list?

Won't Django raise error on this?

Reply:
Wasn't sure about that and replaced the function with `migrations.RunPython.noop` as suggested